ios/FluentUI/Vnext/Persona/PersonaView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public protocol PersonaCellProtocol {
+    @objc var persona: MSFAvatarState? { get set }
+    @objc var titleTrailingAccessoryView: UIView? { get set }
+    @objc var subtitleTrailingAccessoryView: UIView? { get set }
+    @objc var onTapAction: (() -> Void)? { get set }
```

Comment:
Can we add an action on the demo controller to demonstrate it? (it can be just like the other list actions)